services/tenant-ui/frontend/src/plugins/i18n/locales/en.json

```diff
@@ -429,6 +429,8 @@
   },
   "session": {
     "countdown": "You are going to be logged out in { seconds } seconds...",
-    "header": "Session timing out"
+    "countdownHeader": "Session timing out",
+    "timeoutMessage": "Your session has timed out due to inactivity and you have been logged out. Please re-enter your login information.",
+    "timeoutHeader": " "
```

Comment:
Better way to have a blank header? Was having trouble figuring one out.

Reply:
Hmm, haven't needed a blank field so nothing that I know.